auto-complete.js

```diff
@@ -117,50 +126,69 @@ var autoComplete = (function(){
             }
             that.parentElement.appendChild(that.sc);
 
-            live('autocomplete-suggestion', 'mouseleave', function(e){
+            live('autocomplete-suggestion', 'mouseleave', function (e) {
                 var sel = that.sc.querySelector('.autocomplete-suggestion.selected');
-                if (sel) setTimeout(function(){ sel.className = sel.className.replace('selected', ''); }, 20);
+                if (sel) setTimeout(function () { sel.className = sel.className.replace('selected', ''); }, 20);
             }, that.sc);
 
-            live('autocomplete-suggestion', 'mouseover', function(e){
+            live('autocomplete-suggestion', 'mouseover', function (e) {
                 var sel = that.sc.querySelector('.autocomplete-suggestion.selected');
                 if (sel) sel.className = sel.className.replace('selected', '');
                 this.className += ' selected';
             }, that.sc);
 
-            live('autocomplete-suggestion', 'mousedown', function(e){
+            live('autocomplete-suggestion', 'mousedown', function (e) {
                 if (hasClass(this, 'autocomplete-suggestion')) { // else outside click
                     var v = this.getAttribute('data-val');
+                    if (o.queryHistory) {
+                        if (e.path[1].className === 'local-suggestion-remove-button') {
+                            var removeButton = document.querySelector('.local-suggestion-remove-button')
+                            this.removeChild(removeButton);
+                            var text = this.textContent.trim();
+                            this.remove();
```

Comment:
[ChildNode.remove()](https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove) is not compatible with any IE version, avoid using it.
```suggestion
                            this.parentElement.removeChild(this);
```